app/controllers/decidim/participations/admin/copy_participations_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Decidim
+    module Participations
+      module Admin
+        # This controller allows admins to answer participations in a participatory process.
+        class CopyParticipationsController <  Decidim::Admin::Features::CustomBaseController
```

Comment:
I used here a custom parent class as Decidim by default make a `authorize! :manage, current_feature`
which make it impossible for non manager to create resources like copying